### ISSUE TITLE
HTTP API issues - BzMiner 7.2.3

### ISSUE THREAD

**Issue by blackmennewstyle** [bzminer/bzminer#38]
I found one little issue with the HTTP API. I am running the following command line `/home/ceedii/bzminer_v7.2.3_linux/bzminer -a alph -p stratum+tcp://pool.sg.woolypooly.com:3106 -w ********************************.calypso --http_enabled 1 --http_address 192.168.0.19 --http_port 4450 --http_password calypso --nc 1 --disable 10:0 --no_watchdog` on GNU/Linux Ubuntu 20.04.3 using **bzminer_v7.2.3_linux_cuda_tk.tar.gz**, it's an all AMD GPU rig.

According to your own documentation:
```
/help

    http://localhost:4014/help
    Method: GET
    Response Type: text/html
    Parameters: None

HTTP API documentation. The only HTTP endpoint that does not require a password.
```
Well, if i try to follow the instruction and access the help page without the password parameter `http://192.168.0.19:4450/help`, i'm getting this error:
```
{"result": "incorrect password"}
```
Let me know if you want me to do more digging in the HTTP API and find more issues :)

**Comment by bzminer** [comment]
Ah that's a bug, non modifying apis should be the only endpoints that require a password. I've neglected the http api last few releases, so I will go back and test that and report back here. Will have a new release soon so let me know if any other bugs you find so I can get them in, thank you!

**Comment by blackmennewstyle** [comment]
Sorry for the late reply, i did not find anything so far but will definitely report if i find anything.
Enjoy your weekend.

**Comment by blackmennewstyle** [comment]
Issue is definitely solved with `7.2.4` :)
